src/test/java/org/ethereum/beacon/discovery/storage/LocalNodeRecordStoreTest.java

```diff
@@ -34,8 +35,10 @@ public Update(NodeRecord oldRec, NodeRecord newRec) {
     List<Update> listenerCalls = new ArrayList<>();
     LocalNodeRecordStore recordStore =
         new LocalNodeRecordStore(
-            nodeRecord1, nodePair1.getPrivateKey(), (o, n) -> listenerCalls.add(new Update(o, n)));
-
+            nodeRecord1,
+            nodePair1.getPrivateKey(),
+            (o, n) -> listenerCalls.add(new Update(o, n)),
+            (o, n) -> Optional.of(n));
```

Comment:
nit: Can just use the `NOOP` instance here I think.

Reply:
Done.